src/Connection/Cluster/RedisCluster.php

```diff
@@ -75,6 +79,9 @@ public function __construct(
         $this->connections = $connections;
         $this->strategy = $strategy ?: new RedisClusterStrategy();
         $this->slotmap = new SlotMap();
+
+        $this->replicaSlotmap = new SlotMap();
```

Comment:
Why should we use another SlotMap if we have a SlotMap above?

Reply:
The current slot map only tracks primaries and does not seem to have functionality for tracking lists of connections per slot. 
I did not want to introduce too many changes at once